specification/communication/data-plane/PhoneNumbers/preview/2023-10-01-preview/phonenumbers.json

```diff
@@ -1238,6 +1611,10 @@
         "searchId": {
           "description": "The search id.",
           "type": "string"
+        },
+        "consentToNotResellNumbers": {
```

Comment:
This reads very confusingly. They consent NOT to resell? Seems they should consent to resell, thus defaulting to false for privacy. Regardless of privacy defaults, just the wording alone is very confusing. What would a UI display string look like?